src/primitives/torus.jl

```diff
@@ -0,0 +1,54 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Torus(majorRadius, minorRadius)
```

Comment:
Please follow the code style of the project. We don't use camel case for the variable names.